examples/sparse_finch.ipynb

```diff
@@ -42,9 +42,7 @@
     "\n",
     "import numpy as np\n",
     "import scipy.sparse as sps\n",
-    "import scipy.sparse.linalg as splin\n",
-    "\n",
-    "assert sparse.BackendType.Finch == sparse.BACKEND"
```

Comment:
Why do we want to remove that assertion?

Reply:
Because changing back ends is a private API, and using private API in an example would encourage its use.

Reply:
Sure!